### ISSUE TITLE
Instructor's Page Link Broken

### ISSUE THREAD

**Issue by paulinewhite** [bnmnetp/runestone#631]
Logged in as an instructor.  Clicked Instructor's Page.  Gave page "We recently reorganized....go to Instructor's Page...."  Clicking on the Instructor's Page link here gives same page.

**Comment by bnmnetp** [comment]
Hi,

I am seeing no problem like that.  What course are you registered for?

Please see this link:  http://interactivepython.org/runestone/static/help/bugreports.html for other information that will help me track this down.

Thanks,

Brad

**Comment by paulinewhite** [comment]
Dear Brad,

I am accessing
http://interactivepython.org/runestone/static/JavaReview/index.html for the
Interactive Java Review course.  I am using Google Chrome on a Mac Book Pro
with the user name pwhite

 Attached is a screenshot of the error page I receive after logging in.
However, after logging in, I can reload the link above and access the
content, which seems to work fine.  Then, when I click the "head" icon and
select Instructor's page, I get the same error as in the attached
screenshot.

Sincerely,

Pauline



On Mon, Aug 24, 2015 at 2:30 PM, Brad Miller <notifications@github.com>
wrote:

> Hi,
>
> I am seeing no problem like that. What course are you registered for?
>
> Please see this link:
> http://interactivepython.org/runestone/static/help/bugreports.html for
> other information that will help me track this down.
>
> Thanks,
>
> Brad
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bnmnetp/runestone/issues/631#issuecomment-134329191>.
>



-- 

Pauline White
Burnt Hills-Ballston Lake High School
Computer Science Department Representative
NYS Math 7-12 and Technology K-12 Teacher
(518) 399-9141 ext. 83135
Email: pwhite@bhbl.org

**Comment by bnmnetp** [comment]
Unfortunately, I am not seeing an attachment.

But, I think I **may** have figured out what is happening.  We removed the JavaReview-RU course and replaced it with just JavaReview.    I have updated the registration database so that those who were registered for JavaReview-RU are moved over to this new class.

Hopefully this will fix it for you...

Brad

**Comment by paulinewhite** [comment]
Dear Brad,

Here is the attachment, which did not carry over in direct email reply.
![screen shot 2015-08-24 at 2 39 35 pm](https://cloud.githubusercontent.com/assets/7974957/9449906/9e7899dc-4a73-11e5-9909-1b565cab78b7.png)

I logged out and back in, and I am still getting the error.  As well, click on "Our homepage" in the error message gives the same error message.

Sincerely,

Pauline

**Comment by bnmnetp** [comment]
Can you check your address bar and paste in the URL you are sent to after logging in.

I'm assuming that going to http://interactivepython.org/runestone/static/JavaReview/index.html works just fine.

Brad

**Comment by paulinewhite** [comment]
I am/was using the link you assume.  Hmmm....Maybe it took a minute for the changes to carry over?  Seems to be working now.

Thank you.

Pauline

**Comment by bnmnetp** [comment]
Great!

So, what happened was that you were hit with the old double whammy.   We deleted the old out of date JavaReview-RU course, and put in place  system to keep it updated better.

Secondly, I added some code last week to prevent students from seeing the instructor page, unless they were students from one of the "template" courses like thinkcspy, or pythonds.  I forgot to add JavaReview to the list of template courses.  So you were constantly being redirected back the course that did not exist.  Whenever you tried to access the instructor page.

Brad
